app/views/topical_events/show.html.erb

```diff
@@ -34,16 +34,18 @@
       </div>
     <% end %>
 
-    <%= render "govuk_publishing_components/components/govspeak", {
-      } do %>
-        <%= @topical_event.body&.html_safe %>
-    <% end %>
+    <div class="topical-events__margin-bottom">
+      <%= render "govuk_publishing_components/components/govspeak", {
+        } do %>
+          <%= @topical_event.body&.html_safe %>
+      <% end %>
 
-    <% if @topical_event.about_page_link_text %>
-      <p class="govuk-body">
-        <%= link_to(@topical_event.about_page_link_text, @topical_event.about_page_url, html_options: { class: "govuk-link" }) %>
-      </p>
-    <% end %>
+      <% if @topical_event.about_page_link_text %>
+        <p class="govuk-body">
+          <%= link_to(@topical_event.about_page_link_text, @topical_event.about_page_url, html_options: { class: "govuk-link" }) %>
+        </p>
+      <% end %>
+    </div>
```

Comment:
Hmm we do seem to be adding this extra margin a lot - how does similar pages like organisation resolve it? I'm just wondering if we've started digging ourselves a hole where we're fighting the basic layout - but could be that the basic layout just isn't great for a hub style page.

Reply:
The organisations page uses a lot of the class `organisation__margin-bottom` to add spacing, so I don't think we're being inconsistent here. But again, I will defer to anybody with better frontend knowledge than me on the best thing to do.

Reply:
I am inclined to agree with @BeckaL - I think it's good to be consistent with the rest of the application. I am also in the process of doing a write-up to suggest that we discontinue using the [override classes](https://design-system.service.gov.uk/styles/spacing/#using-the-override-classes) as they use `!important` as part of their css. It is better to use the [static spacing mixins](https://design-system.service.gov.uk/styles/spacing/#using-the-static-spacing-function) than to use these classes.

Reply:
Depending on what you need, you might want to use the responsive helpers instead:

```
@include govuk-responsive-padding(6);
@include govuk-responsive-margin(6, "bottom");
```

Reply:
Just to add additional context, the usage of these helper classes was raised in a GOVUK Frontend community meeting (in 2019) where it was decided that we would avoid using override classes and use a single class instead.

Some things to consider might be that rather than having a class specifically for margin (`organisation__margin-bottom`) it should be describing what the element is, ie. `organisation__content-body`.